libs/clients/health-directorate/src/lib/healthDirectorateClient.service.ts

```diff
@@ -137,6 +150,57 @@ export class HealthDirectorateClientService {
     return result
   }
 
+  async getHealthCareLicensesForWorkPermit(
+    auth: Auth,
+  ): Promise<StarfsleyfiUmsoknStarfsleyfi[]> {
+    const licenses = await this.umsoknStarfsleyfiApiWith(
+      auth,
+    ).umsoknStarfsleyfiStarfsleyfiGet()
```

Comment:
I'm fine with 404 throwing here. I can't process the application if Health Directorate errors on licenses

Reply:
> I'm fine with 404 throwing here.
I'm _not_ fine with this. Expected errors should be handled and logged at most as `warning`. Invalid user input is by HTTP standards `4xx` class errors, i.e. `warning`.


Additionally, our monitoring team is losing sleep because non-actionable "errors" are waking them up.

Reply:
I'll add catching the 404

Reply:
I'll add catching and logging